Android/app/build.gradle

```
@@ -58,8 +58,7 @@ android {
 }
 
 dependencies {
-    implementation("live.ditto:ditto:4.4.4")
-
+    implementation("live.ditto:ditto:4.5.0")
```

We just released 4.5.1

```suggestion
    implementation("live.ditto:ditto:4.5.1")
```